File: remove.js
 function removeDuplicate(arr){
    let set = new Set(arr);
    let uniqueArr = Array.from(set);
    for (let i =0; i< uniqueArr.length ; i++) {
        arr[i] = uniqueArrlength[i];
    }
    return uniqueArr.length;
 }

 const arr = [1,1,2,2,2,3,3];
 const k = removeDuplicate(arr);

 console.log("The array after removing the duplicate elements is :");
 for (let i = 0 ; i < k ; i++) {
    console.log(arr[i]);
 }

File: remove.java
import java.util.*;

public class remove {
    public static void main(String[] args) {
        int arr[] = {1,1,1,1,,2,3,1,2,3,,2,4,5,6};
        int k = removeDuplicates(arr);
        System.out.println("The array after removing duplicate element is ");
        for (int i = ; i < k ; i++) {
            System.out.println(arr[i] + " ");
        }
    }
}
static int removeDuplicates(int[] arr) {
    HashSet <integer> set = new HashSet <> ();
        for ( int i = ; i <arr.length ; i++){
            set.add(arr[i]);

        }
        int k = set.size();
        int j = 0;
        for (int x : set){
            arr[j++] = x;

        }
        return k;
    }
}